tests/conftest.py

```diff
@@ -174,3 +174,30 @@ def df_normal_dist():
     df.columns = ["var"]
 
     return df
+
+
+@pytest.fixture(scope="module")
+def df_ciclycal_trans():
```

Comment:
These dataframes will solely be used for the cyclical transformer, so instead of in the config, let's put them in the same test.py file with the tests.